bin/proxy.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-console */
+require('babel-register');
+
+const http = require('http');
+
+const config = require('config');
+const httpProxy = require('http-proxy');
+
+
+const proxy = httpProxy.createProxyServer({ secure: false });
+const apiHost = config.get('proxyApiHost', null) || config.get('apiHost');
+const frontendHost = `http://${config.get('serverHost')}:${config.get('serverPort')}`;
+
+console.log(`apiHost: ${apiHost}`);
+console.log(`frontendHost: ${frontendHost}`);
+
+function getHost(req) {
+  if (!req.headers['user-agent'].includes('Android') || req.url.startsWith('/api/')) {
```

Comment:
Hmm, I guess relying on user agent is ok for now. We'll definitely have to solve it when we start on the desktop site though.

My main concern was developing in Chrome (which is needed for source map support). I just found their responsive mode. It's a little clunky but I guess it will do: https://developers.google.com/web/tools/chrome-devtools/device-mode/